sormas-rest/src/main/java/de/symeda/sormas/rest/exception/mappers/ConstraintViolationExceptionMapper.java

```diff
@@ -1,6 +1,6 @@
 /*
  * SORMAS® - Surveillance Outbreak Response Management & Analysis System
- * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * Copyright © 2016-2022 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
```

Comment:
If the year is already adjusted it should be consistently 2023. This is only an example, please change completely for all touched classes.